airbyte/_util/api_util.py

```
@@ -720,16 +738,178 @@ def delete_connection(
+            context={
+                "workspace_id": workspace_id,
+                "actor_id": actor_id,
+                "actor_type": actor_type,
+                "response": response,
+            },
+        )
+
+
+def check_connector(
+    *,
+    actor_id: str,
```

Ruff (0.8.0)

Unused function argument: `actor_id`

(ARG001)